include/dlr_common.h

```diff
@@ -52,14 +51,12 @@ std::string GetParentFolder(const std::string& path);
 void LoadJsonFromFile(const std::string& path, nlohmann::json& jsonObject);
 
 inline bool StartsWith(const std::string& mainStr, const std::string& toMatch) {
-  return mainStr.size() >= toMatch.size() &&
-  mainStr.compare(0, toMatch.size(), toMatch) == 0;
+  return mainStr.size() >= toMatch.size() && mainStr.compare(0, toMatch.size(), toMatch) == 0;
 }
 
 inline bool EndsWith(const std::string& mainStr, const std::string& toMatch) {
-  if (mainStr.size() >= toMatch.size() &&
-      mainStr.compare(mainStr.size() - toMatch.size(), toMatch.size(),
-                      toMatch) == 0)
+  if (mainStr.size() >= toMatch.size() && \
```

Comment:
`\` is not needed for extending line in C++

Reply:
ran clang-format -style="google". Seem to have fixed it.